instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/GrpcRequest.java

```diff
@@ -17,6 +17,8 @@
 
   @Nullable private volatile SocketAddress remoteAddress;
 
+  @Nullable private volatile String authority;
```

Comment:
This does not change during this object's life, it's set once just after the construction - can you make it a final field instead?